### ISSUE TITLE
Wishlists query with user_id returns empty even though db has value

### ISSUE THREAD

**Issue by prionkor** [yithemes/yith-woocommerce-wishlist#10]
I am doing a very simple query with `userId` which suppose to return wishlist objects matching the user_id. 
```
try {
	$results = \WC_Data_Store::load( 'wishlist' )->query( [ 'user_id' => $user_id ] );
} catch( \Exception $e ){
	// return error response
	return new \WP_REST_Response(array('status' => 500, 'error' => $e->getMessage() ), 500);
}

var_dump($results);
```
Any ideas?

**Comment by alarocca130** [comment]
**TL;DR**
You need to supply `session_id` among query parameters, to avoid system searching for current session

**Description of the problem**
Becuase of default parameters of `\YITH_WCWL_Wishlist_Data_Store::query` method, system will try to retrieve wishlists matching current session, if you're not a logged in user

Since you're also supplying a `user_id`, query won't match any wishlist object when unauthenticated
The easy fix is to always explicitely provide `session_id` parameter, initializing it to `false` in order to automatically exclude any additional query over session id

Here an example of reviewed code:
```
try {
	$results = \WC_Data_Store::load( 'wishlist' )->query( [ 'user_id' => $user_id, 'session_id' => false ] );
} catch( \Exception $e ){
	// return error response
	return new \WP_REST_Response(array('status' => 500, 'error' => $e->getMessage() ), 500);
}

var_dump($results);
```

Let me know if this solves your problem

**Comment by prionkor** [comment]
Thanks, I got it!
